client/components/dashboard/dashboard-service.js

```diff
@@ -777,8 +777,21 @@ DashboardService.prototype.removeContainer = function(container) {
 
 /**
  * Unselect the currently selected widget and container, if any.
- */
-DashboardService.prototype.unselectWidget = function() {
+ *
+ * @param {Event=} opt_event;
+ */
+DashboardService.prototype.unselectWidget = function(opt_event) {
```

Comment:
Recommend moving the event logic to a lower-level function, like "OnDashboardClick".